sale_order_line_input/models/res_config_settings.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    show_view_sale_order_line = fields.Boolean(
```

Comment:
@trisdoan  I'm wondering why not using `group_...` field with `implied_group` instead of this (to avoid the `set_value()` stuff ?